dipy/workflows/utils.py

```diff
@@ -0,0 +1,27 @@
+from os.path import join, dirname, isabs, exists
+from os import makedirs
+
+
+def choose_create_out_dir(out_dir, root_path):
+    if out_dir == '':
+        result_path = dirname(root_path)
+    elif not isabs(out_dir):
+        result_path = join(dirname(root_path), out_dir)
+        if not exists(result_path):
+            makedirs(result_path)
+    else:
+        result_path = out_dir
+
+    return result_path
+
+def int_param(val):
+    return int(val) if val != 'None' else None
+
+def bool_param(val):
+    return (val == 'True') if val != 'None' else None
+
+def int_list_param(val, sep=' '):
+    if val == 'None':
+        return None
+
+    return val.split(sep)
```

Comment:
Actually, these were leftovers from the first draft we had with all parameters as string. They are now removed.